kymatio/scattering2d/backend/torch_skcuda_backend.py

```diff
@@ -152,7 +152,7 @@ def __call__(self, x):
         if not x.is_cuda:
             raise TypeError('Use the torch backend (without skcuda) for CPU tensors.')
 
-        out = x.new(x.shape)
+        out = x.new(x.shape[:-1] +(1,))
```

Comment:
`torch.zeros_like(x[..., :1])`

Reply:
maybe `empty_like` if you are sure you are going to overwrite everything

Reply:
Why the :-1:? Is `new` bad?

Reply:
`zeros_like` or `empty_like`, depending on whether everything gets overwritten, is compatible with numpy syntax, `new` is not

Reply:
the problem is the shape has a 2 at the end, when we want a one.

Reply:
I suppose I can have it use `torch.zeros` and input in all the tensor layouts and tensor devices, but thats a long line of code.